### ISSUE TITLE
S3262 FN: Mark comments as compliant in UTs 

### ISSUE THREAD

**Issue by cristian-ambrosini-sonarsource** [SonarSource/sonar-dotnet#6389]
### Description

FN for S3262 (MethodOverrideNoParams).

Using C#11 static virtual/abstract methods in interfaces, the rule is not raising an issue.

### Repro steps

```csharp
            public interface IMath
{
    static virtual void Method(params int[] numbers) { }
    static virtual void Method(string s, params int[] numbers) { }
    static abstract void Method(string s, string s1, params int[] numbers);
}

public class Foo : IMath
{
    public static void Method(int[] numbers) { } // Compliant, FN
    public static void Method(string s, int[] numbers) { } // Compliant, FN
    public static void Method(string s, string s1, int[] numbers) { } // Compliant, FN
}
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
